app/src/main/java/com/example/yummy/data/source/local/YummyLocalDataSource.kt

```diff
@@ -0,0 +1,47 @@
+package com.example.yummy.data.source.local
+
+import com.example.yummy.data.source.FoodDataSource
+import com.example.yummy.data.source.NoteDataSource
+import com.example.yummy.data.source.local.dao.NoteDao
+import com.example.yummy.data.source.local.utils.LocalAsyncTask
+import com.example.yummy.data.source.local.utils.OnDataLocalCallback
+import com.example.yummy.data.source.model.Note
+
+class YummyLocalDataSource(
+    private val noteDao: NoteDao
+) : NoteDataSource.Local, FoodDataSource.Local {
+    override fun getAllNote(callback: OnDataLocalCallback<List<Note>>) {
```

Comment:
Khi header của class dài nhiều dòng, để cách 1 dòng trống
```suggestion
) : NoteDataSource.Local, FoodDataSource.Local {

    override fun getAllNote(callback: OnDataLocalCallback<List<Note>>) {
```

Reply:
e clear